src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -1197,31 +1199,20 @@ bool CoolingPanelParams::SizeCoolingPanelUA(EnergyPlusData &state)
                           "between the rated temperatures.");
         ShowContinueError(
             state, "If the rated capacity is higher than this product, then the cooling panel would violate the Second Law of Thermodynamics.");
-        SizeCoolingPanelUA = false;
         this->UA = 1.0;
+        return false;
     }
-    if (Tinletr >= Tzoner) {
-        ShowSevereError(state,
-                        format("SizeCoolingPanelUA: Unit=[{},{}] has a rated water temperature that is higher than the rated zone temperature.",
-                               cCMO_CoolingPanel_Simple,
-                               this->EquipID));
+
+    this->UA = -MDotXCp * log(1.0 - RatCapToTheoMax);
+    if (this->UA <= 0.0) {
+        ShowSevereError(
+            state, format("SizeCoolingPanelUA: Unit=[{},{}] has a zero or negative calculated UA value.", cCMO_CoolingPanel_Simple, this->EquipID));
         ShowContinueError(state,
-                          "Such a situation would not lead to cooling and thus the rated water or zone temperature or both should be adjusted.");
-        SizeCoolingPanelUA = false;
-        this->UA = 1.0;
-    } else {
-        this->UA = -MDotXCp * log(1.0 - RatCapToTheoMax);
-        if (this->UA <= 0.0) {
-            ShowSevereError(
-                state,
-                format("SizeCoolingPanelUA: Unit=[{},{}] has a zero or negative calculated UA value.", cCMO_CoolingPanel_Simple, this->EquipID));
-            ShowContinueError(state,
-                              "This is not allowed.  Please check the rated input parameters for this device to ensure that the values are correct.");
-            SizeCoolingPanelUA = false;
```

Comment:
This block was entered before even if we had entered into the error above where the RapCapToTheoMax was > 1.1 I changed the control flow to return early and avoid trying to compute a std::log with a negative value.

Reply:
Seems much more reasonable now to me!